### PR TITLE
HTTP read only endpoints

### LINKED ISSUE #1: Creat read only HTTP endpoints
## Description:
Add a few GET HTTP endpoints to display user and safe information.

## Thing to do:

- [ ] `GET` endpoint to display current user info
- [ ] `GET` endpoint to query the safe

### PR DESCRIPTION
**Pull request by koshikraj**
## Description:
HTTP read-only endpoints to query user and safes.

Run or create a new safient worker by running:

`safient safien worker`

## Screenshots:
safien-worker

worker-http

### CODE REVIEW

File: src/utils/wallet.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export class Wallet {
     // generate(account.address, { small: true });
 
     console.log(info('Retriving account information'))
-    console.log('‍📬 Your address ' + account.address)
+    console.log(info('‍📬 Your wallet address '), account.address)
```
Comment (yathishram) [inline src/utils/wallet.ts:67]: Can we also return DID here so it'll be easy for the user.

Comment (koshikraj) [reply src/utils/wallet.ts:67]: Yes, I have made that update.

Comment (koshikraj) [reply src/utils/wallet.ts:67]: Actually, we are returning the DID in the `connectUser` method.

**Review by yathishram — APPROVED** [review]
LGTM

**Pull request merged**